doc/source/running.rst

```diff
@@ -50,6 +58,26 @@ You can confirm the |pbm-agent| connected to its mongod and started OK by
 confirming *"pbm agent is listening for the commands"* is printed to the log
 file.
 
+How to see the pbm-agent log
+--------------------------------------------------------------------------------
+
+With the packaged systemd service the log output to stdout is captured by
```

Comment:
Add a comma after "service" to make easier to follow - the text has a lot of terminology to consume.

Reply:
I dislike using a comma to indicate 'a person speaking this would pause here'. For me commas are to join semantic units that could be expanded to be their own sentence. E.g. "He looked at his crop, ruined by the hail storm." can be "He looked at his crop. It had been ruined by the hail storm."

Reply:
This is only about English grammar: a propositional phrase at the beginning of a sentence ends with a comma. Your example is about something else (comma operator overloaded ;) - it is used to delimit homogeneous parts of the sentence (a sequence of predicates in your case)

Reply:
I don't know what a propositional phrase is; I'm assuming that it was meant to be prepositional phrase. But prepositional phrases didn't get a comma between them and the object they modify so much.

I think inserting a comma where a person would pause in speech is a growing phenomenon of the last two decades only. It would make sense that rapid text communication ala' internet is reducing the separation in people's mental models of written and spoken language, and think this is one of many side-effects of that.

There were people who liked a 'comma-for-pause' rule since before I was alive, but my memory is they were a definite minority. Because sending words on paper physically raised a bar for what was distributed my sampling is of course weighted towards long, professionally-written text.

I believe the promiscuous comma-for-speech-pausers have then been joined by people who start doing it spontaneously from their computer desktops. Basically everyone does make an effort to self-check when in doubt, but often that doesn't go beyond a check to see if anyone else does it - positive 'confirmation' can thus be taken from minority phenomenon.

FWIW I see confirmation for my memory, that comma use between prepositional phrases and the related object was in minority, in searches on books.google.com. But it is a web search index finding the results naturally, and those are biased in all sorts of directions. And I can't extract the sentences out to plain text so I got worn out clicking through pages before I could count enough to claim a small error factor.

Assuming written language evolution achieves what evolution usually does it means writing will be improved for the new world. But I can't see this increased use of the comma for speech-pause indication will be part of it, except possibly breaking it thoroughly enough for a new grammar mutation to appear and prosper.